com.jetbrains.PyCharm-Professional.appdata.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>com.jetbrains.PyCharm-Professional</id>
+	<metadata_license>CC-BY-SA</metadata_license>
+	<name>PyCharm-Professional</name>
+	<summary>The most intelligent Python IDE</summary>
+	<description>
+		<p>PyCharm is designed by programmers, for programmers, to provide all the tools you need for productive Python development.</p>
+		<p>PyCharm provides smart code completion, code inspections, on-the-fly error highlighting and quick-fixes, along with automated code refactorings and rich navigation capabilities.</p>
+		<p>PyCharm offers great framework-specific support for modern web development frameworks such as Django, Flask, Google App Engine, Pyramid, and web2py.</p>
+		<p>PyCharm integrates with IPython Notebook, has an interactive Python console, and supports Anaconda as well as multiple scientific packages including matplotlib and NumPy.</p>
+		<p>In addition to Python, PyCharm supports JavaScript, CoffeeScript, TypeScript, Cython, SQL, HTML/CSS, template languages, AngularJS, Node.js, and more.</p>
+		<p>A huge collection of tools out of the box: an integrated debugger and test runner; Python profiler; a built-in terminal; and integration with major VCS and built-in Database Tools.</p>
```

Comment:
I know the community edition doesn't have it but I'd like it if we had a disclaimer that this isn't supported by JetBrains here (unless it is!)

Reply:
I don't think it is, and I'm not related to JetBrains in any way.

What should I write in the disclaimer? Should I put it at the beginning or at end of the description?

Reply:
[DataGrip](https://flathub.org/apps/details/com.jetbrains.DataGrip) has a nice little disclaimer at the bottom.

Reply:
Yeah, just like @FakeShemp says, that quick note at the bottom would do fine.